index.js

```diff
@@ -29,7 +29,13 @@ function toJson(value, options) {
   lines = value.split('\n')
 
   if (options.comment) {
-    lines = lines.map(stripComments(options.comment))
+    if (Array.isArray(options.comment)) {
+      for(const comment of options.comment) {
+        lines = lines.map(stripComments(comment))
+      }
+    } else {
+      lines = lines.map(stripComments(options.comment))
+    }
```

Comment:
I’d suggest something like this (as high as possible with other var declarations):

```js
var comments = options.comment
  ? Array.isArray(options.comment)
  ? options.comment : [options.comment]
  : []
```

And then here you can do:

```js
comments.forEach(function (comment) {
  lines = lines.map(stripComments(comment))
})
```

Reply:
Done, thank you!